lib/Dialect/TTIR/Transforms/Passes.cpp

```diff
@@ -886,61 +886,43 @@ class TTIRSplitCompoundLayout
   }
 };
 
-inline uint64_t getElementSizeBytes(Type ty) {
-  uint64_t val = 0;
-  if (isa<TileType>(ty)) {
-    auto tileType = mlir::cast<TileType>(ty);
-    val = tileType.getSizeBytes();
-  } else {
-    val = ty.getIntOrFloatBitWidth() / 8;
-  }
-  return val;
-}
-
-inline uint64_t getMemrefSizeBytes(MemRefType ty) {
-  auto shape = ty.getShape();
-  uint64_t size = getElementSizeBytes(ty.getElementType());
-  return std::accumulate(shape.begin(), shape.end(), size,
-                         std::multiplies<uint64_t>());
-  return size;
-}
-
-inline uint64_t getLayoutMemrefSizeBytes(LayoutAttr layout) {
-  return getMemrefSizeBytes(layout.getMemref());
-}
-
-inline uint64_t getTensorMemrefSizeBytes(RankedTensorType ty) {
-  assert(ty.getEncoding());
-  auto layout = mlir::cast<LayoutAttr>(ty.getEncoding());
-  return getLayoutMemrefSizeBytes(layout);
-}
-
 class TTIRAllocate : public impl::TTIRAllocateBase<TTIRAllocate> {
```

Comment:
In TTIR passes.td could you please update:
![image](https://github.com/user-attachments/assets/68d7165a-4c57-456c-9da4-bbf2020c27b7)